diff --git a/android/build.gradle b/android/build.gradle
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -32,5 +32,5 @@
 dependencies {
     implementation "com.facebook.react:react-android:+"
-    api 'org.jitsi:webrtc:124.+'
+    api 'org.jitsi:webrtc:124.0.0'
     implementation "androidx.core:core:1.7.0"
 }
diff --git a/react-native-webrtc.podspec b/react-native-webrtc.podspec
--- a/react-native-webrtc.podspec
+++ b/react-native-webrtc.podspec
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
   s.libraries           = 'c', 'sqlite3', 'stdc++'
   s.framework           = 'AudioToolbox','AVFoundation', 'CoreAudio', 'CoreGraphics', 'CoreVideo', 'GLKit', 'VideoToolbox'
   s.dependency          'React-Core'
-  s.dependency          'JitsiWebRTC', '~> 124.0.0'
+  s.dependency          'JitsiWebRTC', '= 124.0.2'
 end
